qemu/tests/flow_caches_stress_test.py

```diff
@@ -32,23 +32,32 @@ def run(test, params, env):
     :param params: Dictionary with the test parameters.
     :param env: Dictionary with test environment.
     """
-    msg = "Make sure nf_conntrack is disabled in host and guest."
-    error.context(msg, logging.info)
-    if "nf_conntrack" in utils.system_output("lsmod"):
-        err = "nf_conntrack load in host, skip this case"
-        raise error.TestNAError(err)
+
+    def get_if_queues(ifname):
+        """
+        query interface queues with ethtool -l
+        :param ifname: interface name
+        """
+        cmd = "ethtool -l %s" % ifname
+        out = session.cmd_output(cmd)
+        error_context.context("%s" % out, logging.info)
+        return out
+
+    error_context.context("Make sure nf_conntrack is disabled in host and guest.",
+                          logging.info)
+    if "nf_conntrack" in process.system_output("lsmod"):
+        process.system_output("echo 0 > /proc/sys/net/nf_conntrack_max")
```

Comment:
Command `echo 0 > /proc/sys/net/nf_conntrack_max` appears several times, can be moved to cfg to avoid hardcoding.